internal/cmd/miniooni/libminiooni.go

```diff
@@ -416,10 +416,11 @@ func MainWithConfiguration(experimentName string, currentOptions Options) {
 			OnWiFi:   true, // meaning: not on 4G
 			Charging: true,
 		},
-		InputPolicy:  builder.InputPolicy(),
-		StaticInputs: currentOptions.Inputs,
-		SourceFiles:  currentOptions.InputFilePaths,
-		Session:      sess,
+		ExperimentName: experimentName,
```

Comment:
We need to set the experiment name for experiments using the static-default-input policy